### DISCUSSION TITLE
Resume idea

### DISCUSSION THREAD

**Discussion by raxil2024** [recodehive/Stackoverflow-Analysis#446] [Ideas]
Hey i also wanted to know that.. how good it is to put you academic scores in the CV?

**Comment by SuBroXz11** [comment]
i prefer not to.. But it's personal choice.. you may add it

**Comment by sahil-009** [comment]
[Sahil Mund Rsume.pdf](https://github.com/user-attachments/files/17969361/Sahil.Mund.Rsume.pdf)
if you are a fresher make like this.i am currently using this.
